src/Bundle/test/app/AppKernel.php

```diff
@@ -22,29 +22,40 @@ class AppKernel extends Kernel
      */
     public function registerBundles()
     {
-        return [
+        if ('prod' === $this->getEnvironment()) {
+            $loader = require __DIR__ . '/../../../../vendor/autoload.php';
+            $loader->addPsr4('AppBundle\\', __DIR__ . '/../src/AppBundle/');
+            $loader->addPsr4('Webmozart\\Assert\\', __DIR__ . '/../../../Component/vendor/webmozart/assert/src/');
```

Comment:
I guess it's related to #188 and it should be fixed.

Reply:
@pamil Maybe, we could fix #188 and rebase this one (again).

Reply:
Done in #189 🎉

Reply:
@pamil I rebased (again)